cli/exec.go

```diff
@@ -105,6 +105,7 @@ func ConfigureExecCommand(app *kingpin.Application, a *AwsVault) {
 		BoolVar(&input.Lazy)
 
 	cmd.Flag("stdout", "Print the SSO link to the terminal without automatically opening the browser").
+		Default(os.Getenv("AWS_VAULT_STDOUT")).
```

Comment:
This will output a string. One possible option might be something like: `.OverrideDefaultFromEnvar("AWS_VAULT_STDOUT").Bool()`